modules/desktopsharing/ScreenObtainer.js

```diff
@@ -32,6 +33,8 @@ var firefoxExtInstalled = null;
  */
 var reDetectFirefoxExtension = false;
 
+var eventEmitter = new EventEmitter();
```

Comment:
This should use the eventEmitter from desktopSharing instead of a local one.  See strophe.ping as an example: https://github.com/pstros/jitsi-meet/blob/master/modules/xmpp/strophe.ping.js